### ISSUE TITLE
BUG: "Curse my Baby" curses the wrong kid !

### ISSUE THREAD

**Issue by cordy-123** [jasonrohrer/OneLife#546]
TL;DR Cursing a SID baby using "curse my baby", curses your youngest LIVING child.

--------------------------------------------------------------------

A few days ago, my mom used the 'curse my baby' on a younger sibling (SID-er). She saw me a few moments later and noticed that I had a cursed name. She was very apologetic about it several times. 

I assume she was telling the truth, that she didn't actually curse me specifically; but instead used the "curse my baby" option.

I had not reported the issue since it was the first time I heard of it. And a search of Discord and the OHOL Forum has not turned up anything.

Yesterday, FulmenTheFinn said on the forum:  "I think the mother may have cursed me with "curse my baby" after SID'ing and exiting the game. But I'm unsure, since the command does not seem to target the latest dead baby if you have even one living child."

After maybe six lives trying to be a girl and having a SIDs baby....finally got one and tested it. And indeed, it did curse my youngest living daughter. (Sorry, kid.)

**Comment by jasonrohrer** [comment]
Yes, you're right.  Fixing!
